gitbook-docs/getting-started/installation.md

```diff
@@ -5,7 +5,8 @@ Make sure you've performed the [environment setup](environment-setup.md) before
 continuing with the installation.
 {% endhint %}
 
-The Proof Market toolchain is a collection of Python scripts for interacting with the Proof Market.
+[The Proof Market toolchain](https://github.com/NilFoundation/proof-market-toolchain)
```

Comment:
Should we omit "the" in this case? I feel the "Proof Market" addition defines the toolchain well enough.

Reply:
Ok, fixed